blog/releases/announcing-knative-v0-17-release.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Announcing the Knative v0.17 release"
+linkTitle: "Version 0.17 release"
+date: 2020-08-24
+description: "The Knative v0.17 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+## Announcing Knative v0.17 Release
+
+As Knative maintains its predictable release cadence, it's time to cut another stable release across components. Be aware for the next release we are planning to make Kubernetes 1.17 the minimum version. Serving continues their journey with test and benchmark optimizations on the already v1 API version. For Eventing now ContainerSource and PingSource are available as v1beta1.
+
+Remember to check [the upgrade docs](https://knative.dev/docs/install/upgrade-installation/) for any upgrades concerns applicable to your current version before you upgrade to the new version v0.17
+
+
+### [Serving v0.17.1](https://github.com/knative/serving/releases/tag/v0.17.1)
+
+**InitialScale annotation to control the initial deployment size**
+There is a new annotation that can be used to control the number of pods that are initially deployed when new Revisions are rolled out.
+
+**net-contour and net-kourier have moved to Beta**
+In addition to net-istio, we now have three networking layers that we classify as (at least) Beta.
+
+**Kubernetes minimum version has NOT changed**
+It remains **1.16**, but we may bump to 1.17 in the coming release depending on its availability for qualification testing (per our release principles).
+
+<details><summary>Autoscaling</summary>
```

Comment:
These tags seem odd to me for markdown? What is this used for?

Reply:
Its used on GitHub to create a section that can be collapsed (you can use HTML 5 in GitHub flavored MD).

Reply:
Like @rhuss indicated, this allows for better readability of so much text on the page, allows the user if they want to drill into the details of the pull requests they can click the twisty. while allowing web crawlers to still go thru all the content.

It looks like this 
![image](https://user-images.githubusercontent.com/1094878/91862967-96c66a00-ec3c-11ea-8518-4200b2bebc03.png)